Modules/Typescript/Design-Patterns/pacificParcel.ts

```diff
@@ -0,0 +1,17 @@
+import { ShipperWrapper, ShipmentTypeEnum } from './shipperWrapper';
+
+export class PacificParcel extends ShipperWrapper {
+
+  constructor(weight: number) {
+    super(
+      'Pacific Parcel',
+      ['7', '8', '9'],
+      {
+        [ShipmentTypeEnum.Letter]: 0.51,
+        [ShipmentTypeEnum.Package]: 0.19,
+        [ShipmentTypeEnum.Oversized]: 0.2,
```

Comment:
This is incorrect, you are saying that the formula is .2 * weight, but per task, it's .19 * 160 + .21 * (weight - 160)